_posts/2023-7-12-Add-client-side-HTTP-authentication-mechanism-support.adoc

```diff
@@ -0,0 +1,129 @@
+---
+layout: post
+title: "Upcoming client side HTTP authentication mechanisms support"
+date: 2023-07-12
+tags: http authentication client
+synopsis: The implementation of client side HTTP authentication mechanisms in Elytron is in progress..
+author: kekumar
+---
+
+
+:toc: macro
+:toc-title:
+
+
+This feature will introduce a significant enhancement to WildFly Elytron - the addition of client-side HTTP authentication mechanism support. With this new feature, WildFly Elytron will provide a client-side framework for HTTP authentication mechanisms, allowing for secure and seamless communication with secured server.
```

Comment:
It would be good to mention that this is an upcoming feature.

Reply:
It would be good to provide a summary of the new functionality that's been added to the Elytron authentication client.